ubyssey/static/src/js/components/GallerySlide.jsx

```diff
@@ -3,6 +3,10 @@ var GallerySlide = React.createClass({
         var slideStyle = { width: this.props.width };
         var imageStyle = { backgroundImage: "url('" + this.props.src + "')" };
 
+        if (!this.props.caption) {
+            $('.slide-credit').css('padding-top', '20px')
+        }
+
         var caption = (<p className="slide-caption" dangerouslySetInnerHTML={{__html: this.props.caption}}></p>);
 				var credit = (<p className="slide-credit" dangerouslySetInnerHTML={{__html: this.props.credit}}></p>);
```

Comment:
The indentation is a little off here -- can you fix it to be like the line above?